src/Control/Email/Email.php

```diff
@@ -635,6 +640,15 @@ public function setBody($body)
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function invalidateBody() {
```

Comment:
Our linter wants this to be a separate line 

```suggestion
    public function invalidateBody() 
    {
```

Reply:
Have added a new commit.